jerry-core/api/jerry-debugger.c

```diff
@@ -136,6 +137,9 @@ jerry_debugger_wait_and_run_client_source (jerry_value_t *return_value) /**< [ou
     jerry_debugger_uint8_data_t *client_source_data_p = NULL;
     jerry_debugger_wait_and_run_type_t ret_type = JERRY_DEBUGGER_SOURCE_RECEIVE_FAILED;
 
+    // Notify the client about that the engine is waiting for a source.
+    jerry_debugger_send_type (JERRY_DEBUGGER_IN_WAIT_MODE);
```

Comment:
Hm, I realized that the name is a bit unclear. What about `JERRY_DEBUGGER_WAIT_FOR_SOURCE` ? The `in` is not the perfect phrase since we are about to enter this mode.